lib/models.py

```diff
@@ -0,0 +1,16 @@
+import mongoengine as me
```

Comment:
The as me - is this a common convention?  and not say `import mongoengine as db`?

Reply:
Yes, this is straight from the 'Using MongoEngine with Flask' docs. The `db` variable is reserved in our `lib` package for activating MongoDB. This code, `import mongoengine as me` allows the `models.py` file to have full access to everything included in MongoEngine, but instead of needing to write `mongoengine.StringField()` we can use the `me` to write `me.StringField()`.